doc/whats_new/v0.20.rst

```diff
@@ -545,6 +545,10 @@ Decomposition, manifold learning and clustering
   by :user:`Vighnesh Birodkar <vighneshbirodkar>` and
   :user:`Olivier Grisel <ogrisel>`.
 
+- Fixed a bug in :func:`cluster.k_means_elkan` where the returned `iteration`
+  was missing an iteration. It affected :class:`cluster.KMeans`. :issue:`11353`
```

Comment:
"missing an iteration" reads confusingly. Perhaps "was 1 less than the correct value" is more explicit. Perhaps also mention the `n_iter` attribute.

Reply:
Right I forgot that, thanks.